src/rpc/mining.cpp

```diff
@@ -566,6 +566,8 @@ static RPCHelpMan getblocktemplate()
                     {"segwit", RPCArg::Type::STR, RPCArg::Optional::NO, "(literal) indicates client side segwit support"},
                     {"str", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "other client side supported softfork deployment"},
                 }},
+                {"longpollid", RPCArg::Type::STR, RPCArg::Optional::OMITTED, " "longpollid" of job to monitor for expiration; required and valid only for long poll requests"},
```

Comment:
```suggestion
                {"longpollid", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Delay processing request until the result would vary significantly from the \"longpollid\" of a prior template."},
```

It doesn't necessarily imply the previous template is expired.